sdks/java/build-tools/beam-linkage-check.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
```

Comment:
@iemejia I tried with zsh compatible shell script, but it's not trivial. Therefore this relies on bash.

Reply:
No issues, the shebang should cover it so ok for me